### ISSUE TITLE
Some LaTeX symbols not rendering

### ISSUE THREAD

**Issue by hypnocill** [ShaMan123/react-native-math-view#56]
I'm using MathView to render LaTeX exported from https://cortexjs.io/mathlive/demo/. It works well for the most part but some symbols are not rendering (see screenshot). They appear in the same way in the www.mathjax.org/#demo - red, without proper rendering.
Example troublesome latex would be: \exponentialE\imaginaryI\pi - it renders well in cortexjs but doesn't in react-native-math-view

Is there a way to render those symbols properly? Or to add them to the library myself and instruct the library how to render them, if needed?

Also, a latex with coloring exported from cortejx like so: \colorbox{#fff1c2}{\[ \textcolor{#63b215}{x+42\operatorname{sign}(\placeholder{⬚})} \]}
It results in an error - '\[' is only supported in math mode. If I wrap the whole latex with '$' for math mode - it results in another error - '#' macro is not supported in math mode.

Currently I do: replaceAll(String.raw`\[`, '$') and for the other symbols that don't render like \exponentialE -> e before passing the latex to the mathview which works but seems like a bad workaround.

In conclusion - is there a good way to safely and reliably render the output from https://cortexjs.io/mathlive/demo/ in react-native-math-view without using workarounds like find and replace?

![Screenshot 2021-09-24 at 12 03 59](https://user-images.githubusercontent.com/21131651/134648737-4a96ec23-55d5-4052-a5cf-415ec14a8613.png)
.

**Comment by ShaMan123** [comment]
Hi, you should define macros so mathjax know what to do. 
Also it is likely you need to escape backslashes (`\\`).
Refer to this commit: https://github.com/ShaMan123/react-native-math-view/commit/65565985aebc70a6d11de8aba355cc419e8765c0

**Comment by ShaMan123** [comment]
for example
```tsx
<MathView
   config={{ macros: {
                'placeholder': ''
            }}
/>
```

**Comment by pkra** [comment]
Random bystander here. Both mathlive and mathjax are completely separate systems, even if both use TeX-like syntax (and TeX itself is different still). If you use one tool for authoring and another for rendering, you're bound to run into such issues. 

The relevant docs are http://docs.mathjax.org/en/latest/input/tex/macros/index.html  and https://cortexjs.io/mathlive/reference/commands/. You can diff them and create all missing macros -- or do the community a favor and create a proper MathJax extension.

**Comment by ShaMan123** [comment]
Thanks @pkra for the clear and accurate comment.
Rendering math is tough, lots of workarounds getting this lib working.

**Comment by hypnocill** [comment]
Thank you very much for the answers!
